File: OskUI2/src/app/dtos/healthFacility/create-health-facility.dto.ts
export class CreateHealthFacilityDto {
  healthFacilityTypeId: string = '';
  name: string = '';
  address?: string;
  phoneNumber?: string;
  email?: string;
  taxNumber?: string;
  corporationName?: string;
  observationBedCount: number = 0;
  totalBedCount: number = 0;
  openingDate?: Date;
  upperHealthFacilityId: string = '00000000-0000-0000-0000-000000000000';
  hfStatus: number = 1;
}

File: OskApi/Dtos/HealthFacilities/CreateHealthFacilityDto.cs
using OskApi.Entities.HealthFacilities;

namespace OskApi.Dtos.HealthFacilities
{
    public class CreateHealthFacilityDto
    {

        public Guid HealthFacilityTypeId { get; set; }
        public string Name { get; set; } = null!;
        public string? Code { get; set; }
        public string? Address { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? TaxNumber { get; set; }
        public string? CorporationName { get; set; }
        public int ObservationBedCount { get; set; }
        public int TotalBedCount { get; set; }
        public DateTime? OpeningDate { get; set; }
        public Guid UpperHealthFacilityId { get; set; }
    }
}
